src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -143,7 +142,7 @@ def __init__(self, name: str, address: str, cluster: Cluster = None):
         self._used_sequencers_numbers: list[int] = []
         self._unused_sequencers_numbers: list[int] = []
 
-    def connect(self):
+    def connect(self, cluster: QbloxCluster = None):
```

Comment:
I moved it here so that the modules don't need the cluster information (that will be provided by the `controller` during the connection) to be instantiated and set up, when instead the `controller` needs the modules in order to be instantiated.

Reply:
Actually I'm planning to remove this dependence and let the controller manage the connection of the modules in a different way, but for now this was a patch.